connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorSourceConfig.java

```diff
@@ -88,6 +88,10 @@ public class MirrorSourceConfig extends MirrorConnectorConfig {
     private static final String OFFSET_LAG_MAX_DOC = "How out-of-sync a remote partition can be before it is resynced.";
     public static final long OFFSET_LAG_MAX_DEFAULT = 100L;
 
+    public static final String HEARTBEATS_REPLICATION_ENABLED = "heartbeats.replication" + ENABLED_SUFFIX;
+    private static final String HEARTBEATS_REPLICATION_ENABLED_DOC = "Whether to replicate the heartbeats topics even when the topic filter does not include them.";
```

Comment:
Sorry for coming back to this, just made a last pass and noticed this. Should we add here that the topic filter overrides this flag or do you think it's well known?

Reply:
Added more details to the doc of the config - is this ok, or too detailed?